helios-services/src/main/java/com/spotify/helios/agent/AgentParser.java

```diff
@@ -273,6 +275,11 @@ protected void addArgs(final ArgumentParser parser) {
         .action(storeTrue())
         .setDefault(false)
         .help("If specified, the agent won't write job histories to ZooKeeper.");
+
+    connectionPoolSize = parser.addArgument("--connection-pool-size")
```

Comment:
can you call the flag (and the field names) `--docker-connection-pool-size` instead to be very clear about what the connection pool is for?